libmscore/tremolo.h

```diff
@@ -44,7 +48,8 @@ class Tremolo final : public Element {
       QPainterPath path;
 
       int _lines;       // derived from _subtype
-      TremoloPlacement _tremoloPlacement = TremoloPlacement::DEFAULT;
+      TremoloPlacement _tremoloPlacement;
+      TremoloBeamStyle _beamStyle;
```

Comment:
Better to always initialize. Why is the initialization removed?

Reply:
Don't know... I'll add it back and also initialize `_beamStyle`.

Reply:
Oh I see, it's probably because all other properties are uninitialized. I can add initialization to all of them.